apps/passport-client/src/embedded.ts

```diff
@@ -0,0 +1,34 @@
+import { PCDGetRequest } from "@pcd/passport-interface";
+import { SerializedPCD } from "@pcd/pcd-types";
+
+export enum EmbeddedScreenType {
+  EmbeddedGetRequest,
+  EmbeddedAccessRequest,
+  EmbeddedAddSubscription,
+  EmbeddedInstall
+}
+
+export interface EmbeddedGetRequest {
+  type: EmbeddedScreenType.EmbeddedGetRequest;
+  request: PCDGetRequest;
+  callback: (serialized: SerializedPCD) => void;
+}
+
+export interface EmbeddedAccessRequest {
+  type: EmbeddedScreenType.EmbeddedAccessRequest;
+  origin: string;
+}
+
+export interface EmbeddedAddSubscription {
+  type: EmbeddedScreenType.EmbeddedAddSubscription;
+  feedUrl: string;
+  feedId: string;
+}
+
+export interface EmbeddedInstall {
+  type: EmbeddedScreenType.EmbeddedInstall;
+}
```

Comment:
what does this do? doesn't appear to be used

Reply:
I'll remove it for now. It can come back if we need it!